youtube_dl/extractor/polskieradio.py

```diff
@@ -96,4 +151,6 @@ def _real_extract(self, url):
         title = self._og_search_title(webpage).strip()
         description = strip_or_none(self._og_search_description(webpage))
 
-        return self.playlist_result(entries, playlist_id, title, description)
+        result = self.playlist_result(entries, playlist_id, title, description)
+        result['_type'] = 'multi_video'
```

Comment:
`multi_video` is used for parts of complete video not for videos belonging to an article.